pallets/delegation/src/tests.rs

```diff
@@ -136,17 +136,17 @@ fn create_delegation_direct_root_successful() {
 			// Create delegation to root
 			let (delegation_id, delegation_node) = (
 				get_delegation_id(true),
-				generate_base_delegation_node(hierarchy_root_id, delegate, Some(hierarchy_root_id)),
+				generate_base_delegation_node(hierarchy_root_id, delegate.clone(), Some(hierarchy_root_id)),
 			);
 			let delegation_info = Delegation::calculate_delegation_creation_hash(
 				&delegation_id,
 				&hierarchy_root_id,
 				&hierarchy_root_id,
 				&delegation_node.details.permissions,
 			);
-			let delegate_signature = delegate_keypair.sign(&hash_to_u8(delegation_info));
```

Comment:
S: Seems like we don't need the `delegate_keypair` variable at all anymore, so we can just get rid of it?

Reply:
218f11688d3633c4d3af0737a1c1c1fbd3259df2